vtex/toolbelt
# Use esModuleInterop compiler option

#### What is the purpose of this pull request?
Almost all vtex apps already use `esModuleInterop` by builder-hub imposition, now it's time for toolbelt to use it.

This is a refactor. 

#### How should this be manually tested?

Install this branch:
```
git clone https://github.com/vtex/toolbelt.git && \
cd toolbelt && \
git checkout refact/use-esmoduleinterop-for-real && \
yarn && yarn global add file:$PWD
```
Check if it compiles and passes all tests

#### Types of changes
